VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+Note that validators cannot set commission on block provisions. This is to prevent a continuous shift in distribution from delegators to validators which would increase the concentration of Atoms and ultimately weaken the security of the system.
+
+### How are block provisions distributed?
+
+Block provisions are distributed proportionally to all validators relative to their total stake. This means that even though each validator gains atoms with each provision, all validators will still maintain equal weight.
+
+Let us take an example where we have 10 validators with equal staking power and a commission rate of 1%. Let us also assume that the provision for a block is 1000 Atoms and that each validator has 20% of self-bonded Atoms. These tokens do not go directly to the proposer. Instead, they are evenly spread among validators. So now each validator’s pool has 100 Atoms. These 100 Atoms will be distributed according to each participant’s stake:
+* Validator gets 20 Atoms
+* All delegators get 80 Atoms
+
+Then, each delegator can claim its part of the 80 Atoms in proportion to their stake in the validator’s staking pool. Note that the validator's commission is not applied on block provisions.
+
+### How are fees distributed?
+
+Fees are similarly distributed with the exception that the block proposer can get a bonus on the fees of the block it proposes if it includes more than the strict minimum of required precommits.
+
+When a validator is selected to propose the next block, it must include at least 2/3 precommits for the previous block in the form of validator signatures. However, there is an incentive to include more than 2/3 precommits in the form of a bonus. The bonus is linear: it ranges from 1% if the proposer includes 2/3rd precommits (minimum for the block to be valid) to 5% if the proposer includes 100% precommits. Of course the proposer should not wait too long or other validators may timeout and move on to the next proposer. As such, validators have to find a balance between wait-time to get the most signatures and risk of losing out on proposing the next block. This mechanism aims to incentivize non-empty block proposals, better networking between validators as well as to mitigate censorship.
+
+Let’s take a concrete example to illustrate the aforementioned concept. In this example, there are 10 validators with equal stake. Each of them applies a 1% commission and has 20% of self-bonded Atoms. Now comes a successful block that collects a total of 1025.51020408 Atoms in fees.
+
+First, a 2% tax is applied.
+* 2% * 1025.51020408 = 20.51020408 Atoms go to the reserve pool (the reserve pool funds will be allocated through governance)
```

Comment:
wohuh. reserve pool should probably be mentioned a little earlier - coming out of the blue here

Reply:
Addressed